src/getHumanAge.test.js

```diff
@@ -1,10 +1,27 @@
-'use strict';
+"use strict";
 
-describe('getHumanAge', () => {
-  const { getHumanAge } = require('./getHumanAge');
+describe("getHumanAge", () => {
+  const { getHumanAge } = require("./getHumanAge");
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test("should return 0 human years for 0 pet years", () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test("should return 0 human years for negative pet years", () => {
+    expect(getHumanAge(-10, -10)).toEqual([0, 0]);
+  });
+
+  test("should return 1 human year for first 15 dog/cat years", () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
+  /* eslint-disable */
```

Comment:
Avoid disabling linter, and if there are no way without disabling - disable particular rules